packages/flutter/test/material/dialog_theme_test.dart

```diff
@@ -34,7 +34,47 @@ MaterialApp _appWithAlertDialog(WidgetTester tester, AlertDialog dialog, {ThemeD
 
 final Key _painterKey = UniqueKey();
 
+Material _getMaterialFromDialog(WidgetTester tester) {
+  final StatefulElement widget = tester.element(
```

Comment:
Same as earlier comment about this function

Reply:
Done.